internal/plugin.go

```diff
@@ -209,22 +209,21 @@ func (p *Plugin) applyDefaults(unmarshaledConfig map[string]interface{}) {
 		p.PolicyDefaults.Controls = defaults.Controls
 	}
 
-	// Policy expanders default to true unless explicitly set in the config.
-	// Gatekeeper policy expander policyDefault
-	igvValue, set := getDefaultBool(unmarshaledConfig, "informGatekeeperPolicies")
```

Comment:
It looks like you accidentally deleted this code during your rebase. Could you please fix it?